scanner/scanner.go

```diff
@@ -358,6 +358,14 @@ func (s *Scanner) scanSpecialCharStringAccent() (token.Token, string) {
 			return token.Illegal, ""
 		}
 		s.next() // consume right brace
+	} else if s.ch == ' ' { // handle implicit braces like '\c c'
```

Comment:
Thanks, this looks pretty clean. I don't think there's a more elegant solution here.

Are multiple spaces allowed in this context? Like: `{\c      c}`.  If so, probably worth adding a loop here:

```go
for s.ch == ' ' {
  s.next() // consume space
}
```

Reply:
I've never seen it done, and seeing it annoying me a _little_ bit, but throwing it blank Overleaf project seems to work. I can add a loop there then!